cachey/tests/test_cache.py

```diff
@@ -1,8 +1,17 @@
 import sys
 from time import sleep
 
+import pytest
+
 from cachey import Cache, Scorer, nbytes
 
+from cachey.cache import cost
+
+def test_cost():
+
+    assert cost(200, 500) == 2.5e-09
+    assert cost(0, 500) == 5e-07
```

Comment:
Let's not assert that they have to be these values exactly.  I think that it would be fine if things just worked, and that the cost was relatively low, like maybe 
```suggestion
    assert 0 <= cost(0, 500) <= 100
```